Disable tests that require sandbox/network/fuse connections that fail in portage sandbox.

Updated for 1.21.0 from the 1.20.0 version of this patch.
--- a/tests/meson.build
+++ b/tests/meson.build
@@ -86,13 +86,11 @@ pytest_files = [
   'test_clipboard.py',
   'test_documents.py',
   'test_document_fuse.py',
-  'test_dynamiclauncher.py',
   'test_email.py',
   'test_filechooser.py',
   'test_globalshortcuts.py',
   'test_inhibit.py',
   'test_inputcapture.py',
-  'test_location.py',
   'test_notification.py',
   'test_openuri.py',
   'test_permission_store.py',
